### PR TITLE
Updated src/Api, Instrumentation and test/Api, Instrumentation files …

### PR DESCRIPTION
**Pull request by kyle-lt**
…to use file scoped namespaces

Toward #4640

## Changes

Updated source and test files to use file scoped namespaces for Api and Instrumentation projects.

## Merge requirement checklist

* [X] [CONTRIBUTING](https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/CONTRIBUTING.md) guidelines followed (nullable enabled, static analysis, etc.)
* [X] Unit tests added/updated
* [X] Appropriate `CHANGELOG.md` files updated for non-trivial changes
* [X] Changes in public API reviewed (if applicable)

### CODE REVIEW

File: test/OpenTelemetry.Instrumentation.Grpc.Tests/GrpcServer.cs [hunk 1]
```diff
@@ -98,7 +104,165 @@ public void Configure(IApplicationBuilder app)
                 {
                     endpoints.MapGrpcService<TService>();
                 });
+            }
+After:
```
Comment (utpilla) [inline test/OpenTelemetry.Instrumentation.Grpc.Tests/GrpcServer.cs:108]: Why add these comments?

File: test/OpenTelemetry.Instrumentation.Grpc.Tests/GrpcTestHelpers/ClientTestHelpers.cs [hunk 2]
```diff
@@ -14,11 +14,19 @@
 // limitations under the License.
 // </copyright>
 
+/* Unmerged change from project 'OpenTelemetry.Instrumentation.Grpc.Tests(net6.0)'
```
Comment (utpilla) [inline test/OpenTelemetry.Instrumentation.Grpc.Tests/GrpcTestHelpers/ClientTestHelpers.cs:17]: Same as https://github.com/open-telemetry/opentelemetry-dotnet/pull/4657/files#r1264211305

**Comment by utpilla** [comment]
@kyle-lt It would be good to reduce the scope of the PR by trying to have lesser number of files/ projects.

**Comment by kyle-lt** [comment]
Fair, I’ll handle the rest in smaller batches.  Feel free to decline this PR, and I’ll start fresh next week. Thanks!--Kyle TullyOn Jul 14, 2023, at 5:18 PM, Utkarsh Umesan Pillai ***@***.***> wrote:﻿
@kyle-lt It would be good to reduce the scope of the PR by trying to have lesser number of files/ projects.

**Pull request closed without merging**